src/QirRuntime/test/QIR-tracer/tracer-config.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// TODO: ideally, this file should be generated by the Q# compiler alongside the qir
+
+#pragma once
+
+#include <string>
+#include <unordered_map>
+
+namespace Microsoft
+{
+namespace Quantum
+{
+    using OpId = int;
+    using Time = int;
+    using Duration = int;
+    using LayerId = size_t;
+}
+}
+
+namespace TracerUser
+{
+using OpId = int;

```

Comment:
If this is the same as `Microsoft::Quantum::OpId` then I would recommend to reuse that. It is better to avoid giving different names/definitions to the same entity.

Reply:
A piece of construction debris left behind... cleaned up :)